src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/ExecutableStatementCountCheck.java

```diff
@@ -252,7 +252,6 @@ private static final class Context {
          */
         private Context(DetailAST ast) {
             this.ast = ast;
-            count = 0;
```

Comment:
The field is `int` and is `0` by default and is not `final`. So this assignment is unneceessary.